### DISCUSSION TITLE
How to sequential sending transactions

### DISCUSSION THREAD

**Discussion by jasl** [polkadot-api/polkadot-api#844] [Q&A] [answered]
Sorry, I'm not good at JavaScript.

I have a list of transactions, and I want to send them sequentially. 

It looks like
```
await sendTx1();
await sendTx2();
await sendTx3();
```

However, `tx.signSubmitAndWatch(rootSigner)` is not a `Promise`, so I can't use `await`.
Could someone give me a hint?

PS: I'm writing some ops scripts, not frontend app

**Comment by josepot** [comment, marked as answer]
> However, `tx.signSubmitAndWatch(rootSigner)` is not a `Promise`, so I can't use `await`.

There is [`signAndSubmit`](https://papi.how/typed/tx#signandsubmit), which returns a Promise. So, you can use `await`.
